elastic/synthetics-recorder
# feat: generate NOTICE.txt

part of #265, [#346](https://github.com/elastic/open-source/issues/346)

## Summary
Generate NOTICE.txt listing dependencies used by Synthetics Recorder. Running the script in postinstall hook so whenever new deps are added, we will update the NOTICE.txt too.

## Implementation details
Borrow [script](https://github.com/elastic/apm-agent-rum-js/blob/main/dev-utils/dep-info.js) from 
[apm-agent-rum-js](https://github.com/elastic/apm-agent-rum-js)
- [x] List npm dependencies that will be bundled in the build
- [x] Include chromium license which is not part of npm dependency

## How to validate this change
- Should see `NOTICE.txt`in the repo
- ~Should see extended `NOTICE.txt` that includes platform-specific binaries included in each artifact's resources dir~ : Figured we can simplify this by just including chromium license in NOTICE.txt
